org.gnome.Papers.json

```diff
@@ -71,8 +71,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://poppler.freedesktop.org/poppler-24.06.1.tar.xz",
-                    "sha256": "1e629e8732286c745fbc0b15a3ee591443fb37a2210856e7f3ec38a0fb93ab13",
+                    "url": "https://poppler.freedesktop.org/poppler-24.07.0.tar.xz",
```

Comment:
Is there a reason you chose 24.07 here? The automatically created #5 shows the latest version: 24.09.